locatelcolombia.com,"HTTPSConnectionPool(host='locatelcolombia.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a93d9b50>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='locatelcolombia.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a8ef8ee0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
logic-immo.be,"HTTPSConnectionPool(host='www.logic-immo.be', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.logic-immo.be', port=443): Read timed out. (read timeout=2)"
logicrdv.fr,"HTTPSConnectionPool(host='logicrdv.fr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.logicrdv.fr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
lombardiabeniculturali.it,"HTTPSConnectionPool(host='lombardiabeniculturali.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'lombardiabeniculturali.it' doesn't match either of 'lispa.it', 'www.lispa.it'"")))","HTTPConnectionPool(host='lombardiabeniculturali.it', port=80): Read timed out. (read timeout=2)"
lonestarinspectioncompany.com,"HTTPConnectionPool(host='www.lonestarinspectioncompany.com', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.lonestarinspectioncompany.com', port=80): Read timed out. (read timeout=2)"
looduspere.ee,"HTTPSConnectionPool(host='www.looduspere.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.looduspere.ee', port=443): Read timed out. (read timeout=2)"
loopnet.com,"HTTPSConnectionPool(host='loopnet.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='loopnet.com', port=80): Read timed out. (read timeout=2)"
loot.co.za,"HTTPSConnectionPool(host='loot.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'loot.co.za' doesn't match 'www.loot.co.za'"")))","HTTPSConnectionPool(host='www.loot.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1129)')))"
lossc.co,"HTTPSConnectionPool(host='lossc.co', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='lossc.co', port=80): Read timed out. (read timeout=2)"
lowes.ca,"HTTPSConnectionPool(host='www.lowes.ca', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lowes.ca', port=443): Read timed out. (read timeout=2)"
lowes.com,"HTTPSConnectionPool(host='www.lowes.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lowes.com', port=443): Read timed out. (read timeout=2)"
lsmu.lt,"HTTPSConnectionPool(host='lsmu.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='lsmuni.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
lsmuni.lt,"HTTPSConnectionPool(host='lsmuni.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='lsmuni.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
lsu.lt,"HTTPSConnectionPool(host='lsu.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'lsu.lt' doesn't match either of 'www.lsu.lt', 'journals.lsu.lt'"")))","HTTPSConnectionPool(host='www.lsu.lt', port=443): Read timed out. (read timeout=2)"
lu.se,"HTTPSConnectionPool(host='lu.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))","HTTPSConnectionPool(host='lu.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))"
luattrongtay.vn,"HTTPSConnectionPool(host='luattrongtay.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='luattrongtay.vn', port=443): Read timed out. (read timeout=2)"
luiss.it,"HTTPSConnectionPool(host='luiss.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a61e4490>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='luiss.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a5204160>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
lukoprekyba.lt,"HTTPSConnectionPool(host='www.lukoprekyba.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lukoprekyba.lt', port=443): Read timed out. (read timeout=2)"
lums.edu.pk,"HTTPSConnectionPool(host='lums.edu.pk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='lums.edu.pk', port=443): Read timed out. (read timeout=2)"
lunchtime.co.nz,"HTTPSConnectionPool(host='lunchtime.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='lunchtime.co.nz', port=443): Read timed out. (read timeout=2)"
luts.ee,"HTTPSConnectionPool(host='luts.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a815f4f0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='luts.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a8ade190>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
luximmo.bg,"HTTPSConnectionPool(host='www.luximmo.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.luximmo.bg', port=443): Read timed out. (read timeout=2)"
luxurycarrentalitaly.it,"HTTPSConnectionPool(host='luxurycarrentalitaly.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='luxurycarrentalitaly.it', port=80): Read timed out. (read timeout=2)"
lzs.lt,"HTTPSConnectionPool(host='lzs.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'lzs.lt' doesn't match either of '*.balt.net', 'balt.net'"")))","HTTPConnectionPool(host='lzs.lt', port=80): Read timed out. (read timeout=2)"
m-almada.pt,"HTTPSConnectionPool(host='m-almada.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a5bbe340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='m-almada.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a3152700>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
m-culture.go.th,"HTTPSConnectionPool(host='m-culture.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a1fbf790>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='m-culture.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a64761c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
macmillandictionary.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
macmillanthesaurus.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
madeingreece.news,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
madrid.org,"HTTPSConnectionPool(host='madrid.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a5667d60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='madrid.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a16f1340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mae.ro,"HTTPSConnectionPool(host='mae.ro', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a778f8b0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mae.ro', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a9b9fa30>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
maedchenflohmarkt.de,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
maff.go.jp,"HTTPSConnectionPool(host='maff.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a7cd2d60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='maff.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a6c2a9a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
maga.gob.gt,"HTTPSConnectionPool(host='maga.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'maga.gob.gt' doesn't match '*.maga.gob.gt'"")))","HTTPConnectionPool(host='maga.gob.gt', port=80): Read timed out. (read timeout=2)"
magaseek.com,"HTTPSConnectionPool(host='magaseek.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a010b820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='magaseek.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a0173a60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
magazinauto.com,"HTTPSConnectionPool(host='www.magazinauto.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.magazinauto.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f149f29c760>, 'Connection to www.magazinauto.com timed out. (connect timeout=2)'))"
maghreb.space,"HTTPSConnectionPool(host='maghreb.space', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.maghreb.space', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1129)')))"
maghress.com,"HTTPSConnectionPool(host='www.maghress.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.maghress.com', port=443): Read timed out. (read timeout=2)"
magyarkozlony.hu,"HTTPSConnectionPool(host='magyarkozlony.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='magyarkozlony.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
mahidol.ac.th,"HTTPSConnectionPool(host='mahidol.ac.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mahidol.ac.th', port=443): Read timed out. (read timeout=2)"
mai.gov.pt,"HTTPSConnectionPool(host='mai.gov.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a3268370>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mai.gov.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a79b61f0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mainstand.co.th,"HTTPSConnectionPool(host='mainstand.co.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mainstand.co.th', port=443): Read timed out. (read timeout=2)"
maisbolsas.com.br,"HTTPSConnectionPool(host='maisbolsas.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='maisbolsas.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
makemytrip.com,"HTTPSConnectionPool(host='www.makemytrip.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.makemytrip.com', port=443): Read timed out. (read timeout=2)"
makeuseof.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
malizakladi.si,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mallaky.com,"HTTPSConnectionPool(host='car.mallaky.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='car.mallaky.com', port=443): Read timed out. (read timeout=2)"
mamaibeba.rs,"HTTPSConnectionPool(host='www.mamaibeba.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mamaibeba.rs', port=443): Read timed out. (read timeout=2)"
man-trailer.com,"HTTPSConnectionPool(host='man-trailer.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='man-trailer.com', port=443): Read timed out. (read timeout=2)"
mandela.ac.za,"HTTPSConnectionPool(host='www.mandela.ac.za', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mandela.ac.za', port=443): Read timed out. (read timeout=2)"
mandmdirect.ie,"HTTPSConnectionPool(host='mandmdirect.ie', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mandmdirect.ie' doesn't match either of 'www.mandmdirect.com', 'mandmdirect.com', 'email.mandmdirect.com', 'www.mandmdirect.ie', 'euro.mandmdirect.com', 'mmgw.mmis-p.com', 'resources.mandmdirect.com', 'wwwm.mandmdirect.com', 'services.mandmdirect.com', 'www.mandmdirect.pl', 'www.mandmdirect.dk', 'www.mandmdirect.fr', 'www.mandmdirect.de', 'unsub.email.mandmdirect.com', 'gateway.mandmdirect.com', 'www.mandmdirect.nl', 'www.mandmdirect.at', 'edocs.mandmdirect.com', 'scripts.mandmdirect.com'"")))","HTTPConnectionPool(host='www.mandmdirect.ie', port=80): Read timed out. (read timeout=2)"
mangozero.com,"HTTPSConnectionPool(host='www.mangozero.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mangozero.com', port=443): Read timed out. (read timeout=2)"
mapanicaragua.com,"HTTPSConnectionPool(host='mapanicaragua.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a8e6ac10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mapanicaragua.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a01fc3a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
marcatucuman.com.ar,"HTTPSConnectionPool(host='marcatucuman.com.ar', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='marcatucuman.com.ar', port=443): Read timed out. (read timeout=2)"
mardinlife.com,"HTTPSConnectionPool(host='www.mardinlife.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mardinlife.com', port=443): Read timed out. (read timeout=2)"
marianives.it,"HTTPSConnectionPool(host='www.marianives.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.marianives.it', port=443): Read timed out. (read timeout=2)"
marijampole.lt,"HTTPSConnectionPool(host='marijampole.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='marijampole.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
marionnaud.fr,"HTTPSConnectionPool(host='marionnaud.fr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='marionnaud.fr', port=80): Read timed out. (read timeout=2)"
market-in.gr,"HTTPSConnectionPool(host='market-in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)')))","HTTPSConnectionPool(host='market-in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)')))"
marquartogsoby.dk,"HTTPSConnectionPool(host='marquartogsoby.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a009a2b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='marquartogsoby.dk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a9784880>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
marumura.com,"HTTPSConnectionPool(host='www.marumura.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.marumura.com', port=443): Read timed out. (read timeout=2)"
mas.gov.sg,"HTTPSConnectionPool(host='mas.gov.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f149f6053a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mas.gov.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a7ca81f0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mascus.lt,"HTTPSConnectionPool(host='mascus.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.mascus.lt', port=443): Read timed out. (read timeout=2)"
mascus.lv,"HTTPSConnectionPool(host='mascus.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.mascus.lv', port=443): Read timed out. (read timeout=2)"
maskinisten.net,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mat3am.net,"HTTPSConnectionPool(host='www.mat3am.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mat3am.net', port=443): Read timed out. (read timeout=2)"
matica.hr,"HTTPSConnectionPool(host='matica.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a9a86730>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='matica.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a0238f70>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
matoppskrift.no,"HTTPSConnectionPool(host='www.matoppskrift.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.matoppskrift.no', port=443): Read timed out. (read timeout=2)"
matsaigon.com,"HTTPSConnectionPool(host='www.matsaigon.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.matsaigon.com', port=443): Read timed out. (read timeout=2)"
mauriciofavero.com.br,"HTTPSConnectionPool(host='mauriciofavero.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mauriciofavero.com.br', port=443): Read timed out. (read timeout=2)"
mazeikiai.lt,"HTTPSConnectionPool(host='mazeikiai.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a30c55e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mazeikiai.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a70efc10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mch.cl,"HTTPSConnectionPool(host='mch.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a54c0340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mch.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a9682250>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mcu.es,"HTTPSConnectionPool(host='mcu.es', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a840ffa0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mcu.es', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a871ba90>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mcw.gov.cy,"HTTPSConnectionPool(host='mcw.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a4c12be0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mcw.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a96d7b50>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mdp.edu.ar,"HTTPSConnectionPool(host='mdp.edu.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a777e0a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mdp.edu.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a72508b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
me.gov.ar,"HTTPSConnectionPool(host='me.gov.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'me.gov.ar' doesn't match either of '*.argentina.gob.ar', 'argentina.gob.ar'"")))","HTTPConnectionPool(host='me.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a0aea2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
meb.k12.tr,"HTTPSConnectionPool(host='meb.k12.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f149f5d1ca0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='meb.k12.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a068abe0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mec.es,"HTTPSConnectionPool(host='mec.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1129)')))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mec.pt,"HTTPSConnectionPool(host='mec.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a8606f40>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mec.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a82050a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mecha.cc,"HTTPSConnectionPool(host='mecha.cc', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14a0aeeaf0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mecha.cc', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14a5e537c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
med.tn,"HTTPSConnectionPool(host='www.med.tn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.med.tn', port=443): Read timed out. (read timeout=2)"
medcom.id,"HTTPSConnectionPool(host='www.medcom.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.medcom.id', port=443): Read timed out. (read timeout=2)"
